firmware/common/sonde_packet.cpp

```diff
@@ -27,6 +27,9 @@
 
 namespace sonde {
 
+static uint8_t calibytes[51*16];	//need these vars to survive
```

Comment:
The warning here is removed because the two vars are now declared once in the cpp file and are seen as 'used' by the compiler.
As I said this is compiling fine, the app is starting fine, yet I have to test with a radiosonde signal to see if it's not killing the processing.